src/index.html

```diff
@@ -1,17 +1,192 @@
+          <div class="container">
+            <label>
+              Surname:
+              <input
+                type="text"
+                id="surname"
+                required
+              >
+            </label>
+          </div>
+          <div class="container">
+            <label>
+              Name:
+              <input
+                type="text"
+                id="name"
+                required
+              >
+            </label>
+          </div>
+          <div class="container">
+            <label>
+              How old are You?
+              <input
+                type="number"
+                name="year"
+                id="year"
+                value="12"
+                min="1"
+                max="100"
+                required
+              >
+            </label>
+          </div>
+          <div class="container">
+            <label>
+              Full date of birth:
+              <input
+                type="date"
+                name="date"
+                id="date"
+                min="1979-12-31"
+                max="2010-12-31"
+                required
+              >
+            </label>
+          </div>
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="terms"
+              id="check"
+              required
+            >
+          </label>
+        </fieldset>
+      </div>
+      <div class="group-container">
+        <fieldset>
+          <legend>Registration</legend>
+          <div class="container">
+            <label>
+              E-mail:
+              <input
+                type="email"
+                name="email"
+                id="email"
+                placeholder="email@example.com"
+                required
+              >
+            </label>
+          </div>
+          <label>
+            Password:
+            <input
+              type="password"
+              name="pass"
+              id="pass"
+              required
+            >
+          </label>
+        </fieldset>
+      </div>
+      <div class="group-container">
+        <fieldset>
+          <legend>An interesting fact about you!</legend>
+          <div class="container">
+            Do you love cats?
+            <label>
+              <input
+                type="radio"
+                name="choose"
+                id="yes"
+              >
+                Yes
+            </label>
+            <label>
+              <input
+                type="radio"
+                name="choose"
+                id="no"
+              >
+                No
+            </label>
+          </div>
+          <label>
+            <div class="container">
+              What is your favorite color?
+              <input
+                type="color"
+                name="col"
+                id="col"
+              >
+            </div>
+          </label>
```

Comment:
Do not use block element like div inside inline elements
```suggestion
          <div  class="container">
            <label>
              What is your favorite color?
              <input
                type="color"
                name="col"
                id="col"
              >
            </label>
          </div>
```